bindings/rust/generate/_run_supported_symbol_builds.sh

```diff
@@ -27,7 +27,7 @@ docker run -v "$(pwd)":"$(pwd)" -w "$(pwd)" --rm --platform linux/amd64 rust:lin
 ## arm64 build
 docker run -v "$(pwd)":"$(pwd)" -w "$(pwd)" --rm --platform linux/arm64 rust:linux-arm64 /bin/bash "${SCRIPT_DIR}"/_collect_symbols_build.sh
 
-sort "${SYMBOLS_COLLECT_FILE}" | uniq > "${SYMBOLS_FILE}"
+sort "${SYMBOLS_COLLECT_FILE}" | uniq | grep -v "^_\?bignum_" | grep -v "pqcrystals" > "${SYMBOLS_FILE}"
```

Comment:
so why do we need to exclude bignum and pqcrystal symbols when in this PR we fixed their prefixing?

Reply:
During the Rust build, "rust-bindgen" parses our header files directly -- including the parsing of an *unfiltered* `boringssl_prefix_symbols.h` (b/c it's not going through CMAKE). We need to ensure that the contents of `boringssl_prefix_symbols.h` generated here is consistent with what is used during the actual CMAKE build.

Reply:
ok, I think I don't understand how the whole prefixing for Rust works. But anyway, I thought this PR makes the bignum and Kyber symbols the same as all other symbols in aws-lc. Is that not the case?

Reply:
Ok. The "bignum" symbols are removed from `boringssl_prefix_symbols.h` by CMake because they are not prefixed. The "pqcrystals" symbols are removed because we are handling the prefixing through a different mechanism. We are effectively doing the same thing for the same reasons, but we're doing it at this point because it's easier.